packages/ui/package.json

```diff
@@ -27,7 +27,8 @@
     "./lib/toast/transaction-toast": "./lib/toast/transaction-toast.tsx",
     "./lib/utils": "./lib/utils.ts",
     "./postcss.config.js": "./postcss.config.js",
-    "./styles/*": "./styles/*"
+    "./styles/*": "./styles/*",
+    "./*": "./components/v2/*/index.tsx"
```

Comment:
This will allow us to import v2 components using shorter syntax: `import ValueViewComponent from `@penumbra-zone/ui/ValueViewComponent`

Reply:
this should also be added to the `publishConfig.exports` below

Reply:
suggestion: create an `src` dir and put the components there to not nest it from the `components/v2` – I see that eventually we'll move everything to the v2 style

Reply:
Great idea — this is much cleaner. I've made this change (as well as adding it to `publishConfig`), and added a deprecation README for the v1 components: https://github.com/penumbra-zone/web/pull/1411/commits/f0550479f7fa4853ce16c9a65df6d33571aa0008